File: Application/ARQuizServer/entities/Competitor.php
<?php
/**
 * Created by PhpStorm.
 * User: Martin
 * Date: 13.2.2017
 * Time: 21:24
 */

namespace entities;


class Competitor
{
    private $id;
    private $competitionId;
    private $userId;

    function __construct($id, $competitionId, $userId)
    {
        $this->id = $id;
        $this->competitionId = $competitionId;
        $this->userId = $userId;
    }

    public function getId()
    {
        return $this->id;
    }

    public function getCompetitionId()
    {
        return $this->competitionId;
    }

    public function getUserId()
    {
        return $this->userId;
    }
}

File: Application/ARQuiz/app/src/main/java/com/lepko/martin/arquiz/Entities/Competitor.java
package com.lepko.martin.arquiz.Entities;

import java.util.List;

/**
 * Created by Martin on 25.2.2017.
 */

public class Competitor {

    private int id;
    private int competitionId;
    private List<Integer> answeredQuestions;

    public Competitor(int id, int competitionId, List<Integer> answeredQuestions) {
        this.id = id;
        this.competitionId = competitionId;
        this.answeredQuestions = answeredQuestions;
    }

    public int getId() {
        return id;
    }


    public int getCompetitionId() {
        return competitionId;
    }

    public List<Integer> getAnsweredQuestions() {
        return answeredQuestions;
    }

    public void setAnsweredQuestions(List<Integer> answeredQuestions) {
        this.answeredQuestions = answeredQuestions;
    }

    public void addAnsweredQuestion(int questionId) {
        answeredQuestions.add(questionId);
    }
}
